src/content/roadmap/security/index.md

```diff
@@ -19,21 +21,23 @@ The upgrade from proof-of-work to proof-of-stake began with Ethereum pioneers 
+Reducing the time Ethereum takes to finalize blocks would provide a better user-experience and prevent sophisticated "reorg" attacks where attackers try to reshuffle very recent blocks to extract profit or censor certain transactions. [**Single slot finality (SSF)**](/roadmap/single-slot-finality) is a way to minimize the finalization delay. Right now there are 15 mins worth of blocks that an attacker could theoretically convince other validators to reconfigure. With SSF, there are 0. Users, from individuals to apps and exchanges, benefit from fast assurance that their transactions will not be reverted, and the network benefits by shutting down a whole class of attacks.
 
 <ButtonLink variant="outline-color" to="/roadmap/single-slot-finality/">Read about single slot finality</ButtonLink>
 
 ## Defending against censorship {#defending-against-censorship}
 
-Another facet of security is preventing certain individuals or small groups of validators from becoming too influential. This includes building software that shares validator responsibilities across multiple nodes. This is known as Distributed Validator Technology (DVT). Staking pools are incentivized to use DVT because it allows multiple computers to collectively participate in validation, adding redundancy and fault-tolerance. It also splits validator keys across several systems, rather than having single operators running multiple validators. This makes it harder for dishonest operators to coordinate attacks on Ethereum. Overall, the idea is to derive security benefits by running validators as _communities_ rather than as individuals.
+Decentralization prevents individuals or small groups of validators from becoming too influential. New staking technologies can help to ensure Ethereum's validators stay as decentralized as possible while also defending them against hardware, software and network failures. This includes software that shares validator responsibilities across multiple nodes. This is known as Distributed Validator Technology (DVT). Staking pools are incentivized to use DVT because it allows multiple computers to collectively participate in validation, adding redundancy and fault-tolerance. It also splits validator keys across several systems, rather than having single operators running multiple validators. This makes it harder for dishonest operators to coordinate attacks on Ethereum. Overall, the idea is to derive security benefits by running validators as _communities_ rather than as individuals.
```

Comment:
Had posted a question/comment about capitalization of this term [here](https://github.com/ethereum/ethereum-org-website/pull/9608#discussion_r1129941023)

```suggestion
Decentralization prevents individuals or small groups of validators from becoming too influential. New staking technologies can help to ensure Ethereum's validators stay as decentralized as possible while also defending them against hardware, software and network failures. This includes software that shares validator responsibilities across multiple nodes. This is known as **distributed validator technology (DVT)**. Staking pools are incentivized to use DVT because it allows multiple computers to collectively participate in validation, adding redundancy and fault-tolerance. It also splits validator keys across several systems, rather than having single operators running multiple validators. This makes it harder for dishonest operators to coordinate attacks on Ethereum. Overall, the idea is to derive security benefits by running validators as _communities_ rather than as individuals.
```